server/config/config.properties

```diff
@@ -0,0 +1,39 @@
+# Database persistence model.  Possible values are memory, redis, and dynomite.
+# If ommitted, the persistence used is memory
+#
+# memory : The data is stored in memory and lost when the server dies.  Useful for testing or demo
+# redis : non-Dynomite based redis instance
+# dynomite : Dynomite cluster.  Use this for HA configuration.
+
+db=redis
+
+# Dynomite Cluster details.
+# format is host:port:rack separated by semicolon
+workflow.dynomite.cluster.hosts=mender-redis:6379:us-east-1b
```

Comment:
It's important to have ability to change parameters like this. I believe we already have such pattern used in api-gateway service, when we have some special 'vars' like [@LOGS_FORMAT@](https://github.com/mendersoftware/mender-api-gateway-docker/blob/master/nginx.conf#L35) for instance, which are replaced by entrypoint script. This gives ability to configure service for different environments (dev/staging/prod) by setting environment variables.

Reply:
I do not fully understand what kind of change you are requesting to this PR.
Should this be a part of conductor update?
Or can you create a separate task for it?

Reply:
Yeah, agree, it makes sense to move it to separate task.